colossalai/booster/plugin/hybrid_parallel_plugin.py

```diff
@@ -45,7 +47,23 @@ def __init__(self, module: Module, precision: str, shard_config: ShardConfig, dp
             module = module.to(dtype=torch.bfloat16).cuda()
         else:
             module = module.cuda()    # train without AMP
-        # TODO(ver217): support TP+DP
+
+        if use_ddp:
+
+            ddp_config = dict(process_group=dp_group,
+                              broadcast_buffers=True,
+                              bucket_cap_mb=25,
+                              find_unused_parameters=False,
```

Comment:
`find_unused_parameters=False` and `static_graph=False` may lead to correctness issue when there are unused params. This is commonly observed when layer drop is used.

Reply:
modified